tutorial9_3.R

```diff
@@ -0,0 +1,51 @@
+<<<<<<< HEAD
```

Comment:
Make sure to remove merge conflict text! Your script should be able to run all the way through without any errors and this would certainty cause an error.